### PR TITLE
doc: Improved dashboard documentation

### PR DESCRIPTION
**Pull request by LenzGr**
Moved content (RGW access configuration) from the dashboard `README.rst` into the reference manual. Made various textual improvements to both the README and the documentation.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by LenzGr** [comment]
See http://docs.ceph.com/ceph-prs/21443/mgr/dashboard/ for the rendered version of the updated documentation.

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+Overview
+--------
+
+The original Ceph manager dashboard that was shipped with Ceph "Luminous"
+started out as a simple read-only view into various run-time information and
+performance data of a Ceph cluster. It used a very simple architecture to
+achieve the original goal.
+
+However, there was a growing demand for adding more web-based management
+capabilities, to make it easier to administer Ceph for users that prefer a WebUI
```
[the remark this answers is not in the document]
Comment (LenzGr) [reply doc/mgr/dashboard.rst]: According to the dictionary, "Administer is the verb form for administration or administrator." - "to administrate" is an incorrect form of the verb.

File: src/pybind/mgr/dashboard/README.rst [hunk 2]
```diff
@@ -1,40 +1,35 @@
...
-administrators that prefer a WebUI over the command line.
+However, there was a growing demand for adding more web-based management
+capabilities, to make it easier to administer Ceph for users that prefer a WebUI
+over using the command line.
 
-This module is an ongoing project to add a native web based monitoring and
-administration application to Ceph Manager. It aims at becoming a successor of
-the existing dashboard, which provides read-only functionality and uses a
-simpler architecture to achieve the original goal.
+This new dashboard module is a replacement of the previous dashboard and an
```
Comment (callithea) [inline src/pybind/mgr/dashboard/README.rst]: Suggestion: "[...] of the _previous_ one [...]" (instead of 2x "dashboard" in this sentence).

Comment (LenzGr) [reply src/pybind/mgr/dashboard/README.rst]: Good idea, updated accordingly.

**Comment by LenzGr** [comment]
retest this please

**Comment by rjfd** [comment]
jenkins retest this please

**Pull request merged**